README.md

```diff
@@ -1 +1,138 @@
-# First-Capstone-Project
+
+# First-Captone-Project
+
+# IT Audit
+
+This is a project acknowledging and affirming that I have properly understood everything learnt so far from the first module.
+
+> Learning objectives
+
+- I Used semantic HTML tags.
+- I Applied best practices in HTML code.
+- Used CSS selectors correctly.
+- Used CSS box model.
+- Used Flexbox to place elements on the page.
+- Demonstrated ability to create UIs adaptable to different screen sizes using media queries.
+- Used GitHub Pages to deploy web pages.
+- Applied JavaScript best practices and language style guides in code.
+- Used JavaScript to manipulate DOM elements.
+- Used JavaScript events.
+- Used objects to store and access data.
+- Communicated technical concepts to other technical people.
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+
+Click [here] [https://loom.com/share/f8bb0307b6ee457b8dc2c05882d193e5] to watch a walk-through description, given by me.
+
+## Additional Tools
+
+- WebHint
+- Stylelint
+- ESlint
+- Node Packet Manager
+
+## Live Demo
+
+[Here] [https://loom.com/share/f8bb0307b6ee457b8dc2c05882d193e5] [https://drive.google.com/file/d/1GOQo_9m-o9jGKPHQuGmEziOen_En5Zgu/view?usp=sharing]is the live demo.
```

Comment:
The live demo should be the link to your website. In this instance that should be `https://alabi12.github.io/First-Captone-Project/`. Kindly paste that here since the link the link to the zoom or loom video does not qualify as the live demo. 
I wanted to overlook it and approve your project, :confused:  but I am unable to since there is no link of the live demo of your website anywhere in the readme file. :disappointed:  I can see it has been hosted on github and working perfectly, so kindly paste the link here.